go/config/config.go

```diff
@@ -180,6 +182,7 @@ func (c *CTConfig) Init() {
 	confBool(c.LogExpiredEntries, section, "logExpiredEntries", false)
 	confBool(c.RunForever, section, "runForever", false)
 	confUint64(c.PollingDelay, section, "pollingDelay", 600)
+	confUint64(c.RemoteSettingsUpdateInterval, section, "remoteSettingsUpdateInterval", 60)
```

Comment:
60 seconds seems a bit frequent for this.

Reply:
haha, yeah there's a commit in part 4 that changes it to an hour. I had it at 60s for manual testing.